keras_nlp/tokenizers/word_piece_tokenizer_test.py

```diff
@@ -183,6 +191,30 @@ def test_from_file(self):
         tokenizer = WordPieceTokenizer(vocabulary=vocab_path)
         call_output = tokenizer(input_data)
         self.assertAllEqual(call_output, [[1, 2, 3, 4, 5, 6, 7]])
+    
+    def test_from_file_with_vocabulary_size_arg(self):
+        vocab_path = os.path.join(self.get_temp_dir(), "vocab.txt")
+        input_data = ["the quick brown fox."]
+        vocab_data = ["[UNK]", "the", "qu", "##ick", "br", "##own", "fox", "."]
+        vocab_size = 3
+        with tf.io.gfile.GFile(vocab_path, "w") as file:
+            for piece in vocab_data:
+                file.write(piece + "\n")
+        tokenizer = WordPieceTokenizer(vocabulary=vocab_path, vocabulary_size=vocab_size)
+        call_output = tokenizer(input_data)
+        self.assertAllEqual(tokenizer.vocabulary_size_(), vocab_size)
```

Comment:
just collapse these two test into one, and assert both the layer output and the size of the vocabulary in the same test.